### DISCUSSION TITLE
Github app android preview svg images

### DISCUSSION THREAD

**Discussion by Angelk90** [community/community#107542] [Mobile]
### Select Topic Area

Product Feedback

### Body

It would be nice to be able to have a preview of the svg image, as happens on the github website.

Instead of Github android app for an svg image only the code is shown.

**Comment by elisealix22** [comment]
Hi @Angelk90 , is there any chance you can provide a screenshot or an example of where this is happening? For example, is this happening in a repository description? I'd love to dig in more so we can reproduce it! Thank you for reporting this and I'm sorry you're unable to view the SVG.

**Reply by Angelk90** [reply]
@elisealix22 :

Link: https://github.com/zed-industries/zed/blob/main/assets/icons/arrow_left.svg

**Reply by elisealix22** [reply]
That is extremely helpful, thank you! I added an internal issue to track this and we will investigate. Thanks again for following up so quickly!

**Reply by Angelk90** [reply]
@elisealix22 : The problem also happens when viewing the commit: https://github.com/zed-industries/zed/commit/f19378135acc31b5849be63c55934235fb79ca94

Open the link from both the web and the Android Github app.

As you can see from the Android version of the image below it is displayed this way.

Furthermore, the display of the svg code has a strange behavior, I don't understand why in the final part in the closing tags they are displayed in red as a background, can you tell me the reason?
![Screenshot_20240217-215125_GitHub.jpg](https://github.com/community/community/assets/20476002/5240abe4-f00c-414f-8665-0d6b5cdd9839)

**Reply by Angelk90** [reply]
@elisealix22 : is there anything new?

**Comment by jpmaniqis** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> It would be nice to be able to have a preview of the svg image, as happens on the github website.
> 
> 
> 
> Instead of Github android app for an svg image only the code is shown.

**Comment by 25muA** [comment]
how do I tell Dr Nicolas kokkalis to apply the name change review setting on pi network

**Comment by MatyaGlory** [comment]
Hi
